### PR TITLE
DTSPO-5953 sbox: Migrate MI functions to fluxv2

### PR DESCRIPTION
**Pull request by mokainos**

### CODE REVIEW

File: k8s/namespaces/mi/kustomization.yaml [hunk 1]
```diff
@@ -3,8 +3,8 @@ kind: Kustomization
 namespace: mi
 resources:
 - namespace.yaml
-- mi-azure-functions/mi-azure-functions-identity.yaml
-- mi-azure-functions/mi-azure-functions-identity-binding.yaml
+#- mi-azure-functions/mi-azure-functions-identity.yaml
+#- mi-azure-functions/mi-azure-functions-identity-binding.yaml
```
Comment (timja) [inline k8s/namespaces/mi/kustomization.yaml:7]: is this going to break other environments?

File: k8s/environments/sbox/common-overlay/mi/kustomization.yaml [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
+- ../../../../namespaces/mi/mi-data-extractor/mi-data-extractor-identity.yaml
+- ../../../../namespaces/mi/mi-data-extractor/mi-data-extractor-identity-binding.yaml
+- ../../../../namespaces/mi/mi-extraction-service/mi-extraction-service-identity.yaml
+- ../../../../namespaces/mi/mi-extraction-service/mi-extraction-service-identity-binding.yaml
+- ../../../../namespaces/mi/mi-house-keeping-service/mi-house-keeping-service-identity.yaml
+- ../../../../namespaces/mi/mi-house-keeping-service/mi-house-keeping-service-identity-binding.yaml
+- ../../../../namespaces/mi/mi-integration-service/mi-integration-service-identity.yaml
+- ../../../../namespaces/mi/mi-integration-service/mi-integration-service-identity-binding.yaml
+- ../../../../namespaces/mi/mi-staging-service/mi-staging-service-identity.yaml
+- ../../../../namespaces/mi/mi-staging-service/mi-staging-service-identity-binding.yaml
```
Comment (timja) [inline k8s/environments/sbox/common-overlay/mi/kustomization.yaml:15]: does this need to b e done in other environments if we're changing the base?

File: k8s/environments/sbox/common-overlay/mi/kustomization.yaml [hunk 3]
```diff
@@ -1,8 +1,18 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- ../../../../namespaces/mi
+- ../../../../namespaces/mi/mi-developer-rbac.yaml
```
Comment (timja) [inline k8s/environments/sbox/common-overlay/mi/kustomization.yaml:4]: are these files to be migrated in a follow up PR?

**Review by timja — APPROVED** [review]

File: k8s/environments/sbox/common-overlay/mi/kustomization.yaml (hunk 3 above)
Comment (mokainos) [reply k8s/environments/sbox/common-overlay/mi/kustomization.yaml:4]: yes

**Pull request merged**